.travis.yml

```diff
@@ -135,7 +135,7 @@ script:
 after_script:
   - set -e
   - bash etc/ci-gather-coverage.sh
-  - bash <(curl -s https://codecov.io/bash)
+  - bash <(curl -s https://codecov.io/bash) -f "!./pkg/*" -f "!./extern/*"
```

Comment:
Ah, I did not try that. If this doesn't work, I wonder if the `-g GLOB` option might also be useful?